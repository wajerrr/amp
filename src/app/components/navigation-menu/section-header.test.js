import React from 'react';
import renderer from 'react-test-renderer';
import SectionHeader from './section-header';

describe('SectionHeader', () => {
  it('should match snapshot for a normal headerLink', () => {
    const tree = renderer
      .create(<SectionHeader title="headerSection" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
