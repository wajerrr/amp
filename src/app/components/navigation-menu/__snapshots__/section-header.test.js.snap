// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader should match snapshot for a normal headerLink 1`] = `
<h4
  className={undefined}
>
  <span>
    headerSection
  </span>
  <span
    className={undefined}
  >
    <svg
      version="1.1"
      viewBox="0 0 48 48"
      x="0px"
      xmlSpace="preserve"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <polygon
        points="33.2,18.6 36,21.4 24,33.4 12,21.4 14.8,18.6 24,27.7 	"
      />
    </svg>
  </span>
</h4>
`;
